boards/arm/nucleo_f446re/nucleo_f446re.dts

```diff
@@ -86,6 +87,16 @@
 	status = "okay";
 };
 
+&can1 {
+	pinctrl-0 = <&can1_rx_pb8 &can1_tx_pb9>;
```

Comment:
A comment highlighting the conflict would be nice

Reply:
done